src/Scripting/CoreTestUtilities/ObjectFormatterFixtures/MockDesktopSpinLock.cs

```diff
@@ -37,7 +35,7 @@ public bool IsThreadOwnerTrackingEnabled
 
         internal class SpinLockDebugView
         {
-            private MockDesktopSpinLock m_spinLock;
+            private readonly MockDesktopSpinLock m_spinLock;
```

Comment:
This should have been failing in the current master. This line is *after* restoring IDE0044.

Reply:
Yes, this seems weird indeed. Maybe needs a follow-up investigation..